### DISCUSSION TITLE
Remote sound (mac osx)

### DISCUSSION THREAD

**Discussion by giuliovale** [rustdesk/rustdesk#4585] [General]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Bug Description

Windows 11 -> Windows 11, I hear the sound of the remote computer
Windows -> OSX , the sound of the remote computer is not heard

### How to Reproduce

Simple connect

### Expected Behavior

Hear the sound of the remote computer

### Operating system(s) on local side and remote side

Windows 11 -> OSX

### RustDesk Version(s) on local side and remote side

nightly -> nightly

**Comment by rustdesk** [comment]
Actually not implemented yet since starting the project.
In milestone. https://github.com/rustdesk/rustdesk/discussions/918

**Comment by cyclism6486** [comment]
I'm wondering about this too....Windows 11 PC cannot hear remote macOS sounds, but the other way around works fine (Windows 11 sounds are heard on Mac).  Is this still being looked at?  If not, or at least until it's implemented, is there a way to indicate on the PC end that sounds will not be heard from a remote Mac?

**Comment by godzfire** [comment]
Is this still not implemented? I hope I'm just missing something. I don't understand how Win to Win and Mac to Win would be supported but Win to Mac still isn't. I use my W10 to R/D to my home Mac but can't get any sound.

**Comment by rustdesk** [comment]
This is a good example if you want a feature request get high priority.
https://github.com/rustdesk/rustdesk/discussions/7437

**Comment by j-palindrome** [comment]
Is this supported yet? Expressing interest as a MacOS -> iPad user.
